sql_metadb/derived_tables/openaccess_publication_request.sql

```diff
@@ -0,0 +1,195 @@
+
+COMMENT ON COLUMN openaccess_publication_request.pr_request_status IS 'UUID reference to a refdata value';
+
+COMMENT ON COLUMN openaccess_publication_request.pr_request_status_value IS 'Refdata value for publication request status';
+
+COMMENT ON COLUMN openaccess_publication_request.pr_request_status_label IS 'Refdata label for publication request status';
+
+COMMENT ON COLUMN openaccess_publication_request.pr_pub_type_fk IS 'UUID reference to a refdata value';
+
+COMMENT ON COLUMN openaccess_publication_request.pr_pub_type_value IS 'Refdata value for publication request publication type';
+
+COMMENT ON COLUMN openaccess_publication_request.pr_pub_type_label IS 'Refdata label for publication request publication type';
+
+COMMENT ON COLUMN openaccess_publication_request.pr_authnames IS 'List of names of the authors of the publication';
+
+COMMENT ON COLUMN openaccess_publication_request.pr_corresponding_author_fk IS 'FK for a record in table request_party';
+
+COMMENT ON COLUMN openaccess_publication_request.pr_corresponding_author_role_value IS 'Refdata value for the role of the corresponding author';
+
+COMMENT ON COLUMN openaccess_publication_request.pr_corresponding_author_role_label IS 'Refdata label for role of the corresponding author';
+
+COMMENT ON COLUMN openaccess_publication_request.pr_corresponding_author_name IS 'Name of the linked corresponding author';
+
+COMMENT ON COLUMN openaccess_publication_request.pr_corresponding_author_rp_party_fk IS 'FK to the table party';
+
+COMMENT ON COLUMN openaccess_publication_request.pr_local_ref IS 'ID to use for a local repository or storage';
+
+COMMENT ON COLUMN openaccess_publication_request.pr_pub_url IS 'URL for the publication';
+
+COMMENT ON COLUMN openaccess_publication_request.pr_subtype IS 'UUID reference to a refdata value';
+
+COMMENT ON COLUMN openaccess_publication_request.pr_subtype_value IS 'Refdata value for publication request subtype';
+
+COMMENT ON COLUMN openaccess_publication_request.pr_subtype_label IS 'Refdata label for publication request subtype';
+
+COMMENT ON COLUMN openaccess_publication_request.pr_publisher IS 'UUID reference to a refdata value';
+
+COMMENT ON COLUMN openaccess_publication_request.pr_publisher_value IS 'Refdata value for publication request publisher';
+
+COMMENT ON COLUMN openaccess_publication_request.pr_publisher_label IS 'Refdata label for publication request publisher';
+
+COMMENT ON COLUMN openaccess_publication_request.pr_license IS 'UUID reference to a refdata value';
+
+COMMENT ON COLUMN openaccess_publication_request.pr_license_value IS 'Refdata value for publication request license';
+
+COMMENT ON COLUMN openaccess_publication_request.pr_license_label IS 'Refdata label for publication request license';
+
+COMMENT ON COLUMN openaccess_publication_request.pr_doi IS 'DOI for the publication';
+
+COMMENT ON COLUMN openaccess_publication_request.pr_group_fk IS '';

```

Comment:
Is this going to be a placeholder for future changes?
Or is there another usecase for empty comments?

Reply:
It is a placeholder. We have to clarify this in one of our next working group meetings. The change can then be made very quickly afterwards.